samples/Common.c

```diff
@@ -868,16 +864,18 @@ STATUS sessionCleanupWait(PSampleConfiguration pSampleConfiguration)
     ENTERS();
     STATUS retStatus = STATUS_SUCCESS;
     PSampleStreamingSession pSampleStreamingSession = NULL;
-    UINT32 i;
-    BOOL locked = FALSE;
+    UINT32 i, clientIdHash;
+    ;
```

Comment:
Stray `;`

Reply:
D'oh!. Will fix. Apparently, the clang formatter does smart things :)